packages/frontend-2/components/project/page/stats-block/Settings.vue

```diff
@@ -24,14 +24,20 @@
       </div>
     </template>
     <template #default>
-      <ProjectPageTeamDialog v-model:open="dialogOpen" :project="project" />
+      <ProjectPageTeamDialog
+        v-model:open="dialogOpen"
+        :project="project"
+        :open-section="openSection || undefined"
```

Comment:
Why not just make the ref `ref<OpenSectionType | undefined>()` so that you don't need to do this? The introduction of `null` into the mix doesn't seem to bring any benefit